.gitignore

```diff
@@ -10,3 +10,4 @@ src/pypgstac/target
 src/pypgstac/python/pypgstac/*.so
 .vscode
 .ipynb_checkpoints
+Cargo.lock
```

Comment:
This is up to you; in my projects I include the lockfile even for libraries so that developers at the same commit or checking out the same PR are always using the exact same dependency versions 🤷‍♂️

Reply:
https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html recommends ignoring it for libraries. That same page recommends version-controlling `Cargo.lock` for `cdylib`, so we should keep it.

Reply:
Interesting; I guess `cdylib` making a system library is in effect an application!